pkgs/desktops/cinnamon/mint-artwork/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mint-artwork";
+  version = "1.4.3";
+
+  src = fetchurl {
+    url = "http://packages.linuxmint.com/pool/main/m/mint-artwork/mint-artwork_${version}.tar.xz";
+    sha256 = "126asxpg722qfg2wkwcr7bhsplchq3jn6bkdwf1scpc5za8dd62j";
+  };
```

Comment:
you need to add `glib` to `nativeBuildInputs`. This is because there's a setup-hook that moves glib schemas to share/gsettings-schemas/${pname}-${version}/glib-2.0/schemas